libs/core/src/lib/tile/tile.component.html

```diff
@@ -1,3 +1,12 @@
-<ng-content select="[fd-tile-media]"></ng-content>
-<ng-content select="[fd-tile-content]"></ng-content>
-<ng-content></ng-content>
+<div class="fd-tile" #tileDiv>
```

Comment:
What about putting there `tabindex="0"`? Maybe you can add some flag to tile component, that will control tabindex of this element